### PR TITLE
add additional funding usage to sponsors.html

### PR DESCRIPTION
**Pull request by mjlbach**

### CODE REVIEW

**Review by clason — APPROVED** [review]

**Comment by teto** [comment]
do you have the discourse hosting fee in mind ? just dropping here as with the Internet I expect the usual "what hosting fees, github is free"

**Comment by mjlbach** [comment]
I was thinking security advisory email, discourse, any fees related to neovim.io like domain registration (not sure if we currently have a deal on this), if we have to pay for an element bridge at some point.

**Comment by mjlbach** [comment]
Happy to change the language, we just aren't currently paying for some of these things but *might*, so I didn't want to mislead people that we were spending money on things that are currently free but we may have to pay for due to increasing volume (like the success of the discourse necessitating moving to a paid hosting plan).

File: sponsors.html
Comment (teto) [inline sponsors.html:33]: there are trailing spaces, my OCD kicked in and also as I know nothing about the web I worry it can affect the display xD

**Review by teto — APPROVED** [review]

File: sponsors.html
Comment (mjlbach) [reply sponsors.html]: Fixed, can you squash changes before merging.

**Comment by teto** [comment]
No pb with wording, it's a fair expectation. Feel free to merge when you are happy with it

**Comment by mjlbach** [comment]
I can't merge in neovim.github.io unfortunately, no commit bit

**Pull request merged**
